README.md

```diff
@@ -76,13 +78,39 @@ requests:
   output: yaml
 ```
 
-These two examples are also available in [tst/plans](./tst/plans) directory and can be executed with:
+### Running the Example
+
+This example can be run against the dummy API provided by [tst/server/api.py](./tst/server/api.py)
+
+Start by making sure that `flask` is installed:
+
+```bash
```

Comment:
These could also be `sh` instead of `bash`. Not sure why I have used `bash` for most of these shell snippets as any shell should work.

Reply:
Oh that's also just me. it's not `bash` per say, I really mean `shell`.